design-editor/src/pane/design-editor-element.js

```diff
@@ -619,11 +619,17 @@ class DesignEditor extends DressElement {
 	 */
 	_refreshTAUWidget(id) {
 		const element = this._getElementById(id),
```

Comment:
It probably would be easier to write `const element = this._getElementById(id)[0]` and later refer to it as `element` instead of `element[0]`

Reply:
Done